.bumpversion.cfg

```diff
@@ -1,6 +1,6 @@
 [bumpversion]
 current_version = 0.29.13-alpha
-commit = True
+commit = False
```

Comment:
to make sure I understand, this is false because we are manually creating the commit later on in the workflow yes?

Reply:
@davinchia yup..now we dont want `bumpversion` to commit changes. It would be taken care by the workflow